tests/test_ragaaf.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import unittest
+
+from evals.metrics.ragaaf import AnnotationFreeEvaluate
+
+host_ip = os.getenv("host_ip", "localhost")
+port = os.getenv("port", "8008")
+
+
+class TestRagasMetric(unittest.TestCase):
+
+    # @unittest.skip("need pass localhost id")
+    def test_ragas(self):
+
+        dataset = "explodinggradients/ragas-wikiqa"
```

Comment:
shall we give a simple case to run the ragaaf like ragas? https://github.com/opea-project/GenAIEval/blob/main/tests/test_ragas.py#L34

Reply:
Hi Kaokao, Thank you for sharing this requirement. I added support for unit test cases in our code. Thank you.